metadata.json

```diff
@@ -9,7 +9,7 @@
   "dependencies": [
     {
       "name": "puppetlabs/resource_api",
-      "version_requirement": ">= 1.0.0 < 2.0.0"
+      "version_requirement": ">= 1.6.3 < 2.0.0"
```

Comment:
💯 👍

Reply:
Wait actually, no - this is the resource_api module and not the gem

Reply:
And only 1.0.0 has been released of it (https://forge.puppet.com/puppetlabs/resource_api) so this needs to be changed back :-(